build.gradle.kts

```diff
@@ -19,7 +19,7 @@ plugins {
     // project plugins
     `java-gradle-plugin`
     groovy
-    kotlin("jvm") version "1.4.20"
+    kotlin("jvm") version "1.6.21"
```

Comment:
You might update to Kotlin 1.7

Reply:
In theory, yes, but I am not really familiar and Kotlin 1.6+ was simply the minimal required version.